src/app/data-model-provider/MetadataLookup.cpp

```diff
@@ -82,6 +82,27 @@ std::optional<EndpointEntry> EndpointFinder::Find(EndpointId endpointId)
     return std::nullopt;
 }
 
+Protocols::InteractionModel::Status ValidateClusterPath(ProviderMetadataTree * provider, const ConcreteClusterPath & path,
```

Comment:
This seems to duplicate ValidateClusterPath in EmberMetadata.... Is it worth commoning those up somehow?

Reply:
I think different layers. However the EmberMetadata has no reason to be public anymore, so I will merge that in, hopefully saving some minor flash.